src/utils/extensionizer.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module 'extensionizer' {
+  export default chrome
```

Comment:
what does this do?

Reply:
extensionizer doesn't have types, but it essentially does: `globalThis.chrome || globalThis.browser` https://github.com/MetaMask/extensionizer/blob/main/extension-instance.js#L39-L57. Those APIs are nearly identical https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Chrome_incompatibilities#javascript_apis, so I'm just using chrome's.

That makes all our `extension.tabs.create({` calls typed!

I guess I should explicitly install `@types/chrome` tho.